### PR TITLE
Add request proto to add copy job

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?
Initial commit to add proto to support copy job. Please ignore CopyJob.java since it's incomplete.

### Why are the changes needed?

So we can work on user-facing command and backbone in parallel

### Does this PR introduce any user facing changes?

na

### CODE REVIEW

File: core/server/master/src/main/java/alluxio/master/job/CopyJob.java [hunk 1]
```diff
@@ -0,0 +1,745 @@
...
+      }
+      addLoadedBytes(totalBytes);
+      JOB_LOAD_BLOCK_COUNT.inc(
+          loadTask.getBlocks().size() - response.getBlockStatusCount());
+      JOB_LOAD_BLOCK_SIZE.inc(totalBytes);
+      JOB_LOAD_RATE.mark(totalBytes);
+      return response.getStatus() != TaskStatus.FAILURE;
+    }
+    catch (ExecutionException e) {
+      LOG.warn("exception when trying to get load response.", e.getCause());
```
Comment (ssyssy) [inline core/server/master/src/main/java/alluxio/master/job/CopyJob.java:540]: copy response

File: core/server/master/src/main/java/alluxio/master/job/CopyJob.java [hunk 2]
```diff
@@ -0,0 +1,745 @@
...
+    }
+    CopyJob targetJob = (CopyJob) job;
+    updateBandwidth(targetJob.getBandwidth());
+    setVerificationEnabled(targetJob.isVerificationEnabled());
+  }
+
+  /**
+   * Loads blocks in a UFS through an Alluxio worker.
+   */
+  public class CopyTask extends Task<LoadResponse> {
```
Comment (ssyssy) [inline core/server/master/src/main/java/alluxio/master/job/CopyJob.java:579]: why load?

File: core/server/master/src/main/java/alluxio/master/job/CopyJob.java (hunk 1 above)
Comment (jja725) [reply core/server/master/src/main/java/alluxio/master/job/CopyJob.java:540]: please see PR description that copyjob is still in progress so ignore it.

**Review by ssyssy — APPROVED** [review]

**Comment by jja725** [comment]
@Xenorith mind merging this for me? Thanks

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Comment by jja725** [comment]
alluxio-bot, cherry-pick this to dora please

**Pull request merged**